samples/net/openthread/border_router/prj.conf

```diff
@@ -0,0 +1,101 @@
+CONFIG_EARLY_CONSOLE=y
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_MGMT_EVENT=y
+CONFIG_NET_CONNECTION_MANAGER=y
+CONFIG_NET_NATIVE=y
+CONFIG_NET_ROUTE_MCAST=y
+CONFIG_NET_MAX_MCAST_ROUTES=10
+CONFIG_NET_IPV6_NBR_CACHE=y
+CONFIG_OPENTHREAD_ZEPHYR_BORDER_ROUTER=y
+
+
+# printing of scan results puts pressure on queues in new locking
+# design in net_mgmt. So, use a higher timeout for a crowded
+# environment.
+CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=5000
+CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=16
+
+# Generic OT options
+CONFIG_NET_L2_OPENTHREAD=y
+CONFIG_OPENTHREAD_SHELL=y
+CONFIG_OPENTHREAD_DEBUG=n
+CONFIG_OPENTHREAD_L2_DEBUG=n
+CONFIG_OPENTHREAD_THREAD_VERSION_1_3=y
+CONFIG_OPENTHREAD_L2_LOG_LEVEL_INF=n
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=32768
+CONFIG_SHELL_STACK_SIZE=8192
+CONFIG_OPENTHREAD_THREAD_STACK_SIZE=20480
+CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=16384
```

Comment:
Do we really need such enormous stacks? Please try to optimize (kernel shell can show the actual stack usage).